server/test/views/components/TextFormatterTest.java

```diff
@@ -278,10 +282,8 @@ public void formatTextWithAriaLabel_addsAriaLabel() {
         TextFormatter.formatTextWithAriaLabel(
             "[link](https://www.example.com)", false, false, "test aria label");
 
-    assertThat(content.get(0).render()).contains("aria-label=\"test aria label\"");
-
-    // Set the aria label back to the default for the other tests
-    TextFormatter.resetAriaLabelToDefault();
```

Comment:
Having these reset calls at the end of these tests caused confusing cascading failures when these tests failed. I moved the reset calls to the start of the tests that need them.